layouts/default.vue

```diff
@@ -51,9 +51,10 @@ onMounted(() => {
           content="Radiolab: Podcasts | WNYC Studios | Podcasts"
         />
         <Meta name="og:description" content="Investigating a strange world." />
+        <Meta name="og:image" content="/og-default-image.png" />
         <Meta
-          name="og:image"
-          content="https://media.wnyc.org/i/1200/600/l/80/2020/09/Radiolab_OG_Image_Options_3.png"
+          name="og:image:alt"
+          content="Radiolab: Podcasts | WNYC Studios | Podcasts"
```

Comment:
not sure what og:image:alt is... but should the content be an image? not title text?

Reply:
it's just a default image alt tag - I was testing on facebook and it warned me that it was missing

Reply:
ahh cool